turtleGraphics.js

```diff
@@ -0,0 +1,244 @@
+#! /usr/bin/env node
```

Comment:
i'm seeing a lot of comments throughout. skip comments if the code is self-explanatory. no need to make the file noisy.